README.md

```diff
@@ -0,0 +1,215 @@
+# API Advisor
+
+```json
+"baseURL": "https://csgo-dashboard-rank.herokuapp.com"
+```
+---
+## Summary
+- [Players](#players)
+  - [Register players](#register-players)
+- [Matches](#matches)
+  - [Register matches](#register-matches)
+- [Ranks](#ranks)
+  - [Kills](#rank-kills)
+    - [Display all kills sorted by higher to lower](#rank-kills-sorted)  
```

Comment:
This link leads to nowhere, maybe you can delete this and implement the errors and/or the query params in these cases.